dist/backdraft.js

```diff
@@ -1275,8 +1275,18 @@ _.extend(Plugin.factory, {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var cleanConfigs = this.columnsConfig.map(function(currentValue,index){
+        if (index===0) {
+          return currentValue
+        }
+        if(currentValue.attr&&currentValue.visible){
```

Comment:
is this a net new change or no.. just trying to understand the implementation based on the code that is here... looks like now columns without an `attr` are skipped over, is that true? 

I think we have some tables that have columns with an `attr` key?  If my above assumption is true (and it may not be), why wouldn't we enforce `attr` as a required key up in the column config definition